region/wreckedship/main.json

```diff
@@ -2840,6 +2850,52 @@
                       "shinesparkFrames": 58
                     }}
                   ]
+                },
+                {
+                  "name": "Precise Grapple",
+                  "notable": false,
+                  "requires": [ "canPreciseGrapple" ],
+                  "note": "Fling Samus from the first set of grapple blocks to the second set without falling. Requires precise timing."
```

Comment:
I could go either way on this one. I think it is a similar difficulty to the others, and in Map Rando, I'm expecting `canPreciseGrapple` to be in `Very Hard` difficulty. It is also just to avoid a wall jump and thorn hit.

Reply:
Yeah, a bunch of the other `preciseGrapples` are pretty hard too.